backend/index.js

```diff
@@ -117,19 +114,24 @@ const controller = controllerFactory(
 const oauthApi = oauthApiFactory({ ...config.apis.oauth2 })
 const tokenRefresher = tokenRefresherFactory(oauthApi.refresh, config.app.tokenRefreshThresholdSeconds)
 
-const hmppsCookieOperations = cookieOperationsFactory({
-  name: config.hmppsCookie.name,
-  domain: config.hmppsCookie.domain,
-  cookieLifetimeInMinutes: config.hmppsCookie.expiryMinutes,
-  secure: config.app.production,
-})
+app.use(
+  cookieSession({
+    name: config.hmppsCookie.name,
+    domain: config.hmppsCookie.domain,
+    maxAge: config.hmppsCookie.expiryMinutes * 60 * 1000,
+    secure: config.app.production,
+    httpOnly: true,
+    signed: false,
```

Comment:
`signed` should be `true`, but can't add it in until all apps using cookie session and we've added the key to the environments